front/src/components/FormCreate.vue

```diff
@@ -178,35 +158,25 @@ export default {
                 return this.event.description.length > 140 ? true : false;
             }
         },
-        dateState(){
-            if(this.event.date == null){
+        placeState() {
+            if(this.event.place == null){
                 return null;
-            } else {
-                const date = this.event.date.split("-");
-                if(date < new Date().getFullYear()) {
-                    return true
-                }
+            }else{
+                return this.event.place.length > 5 ? true : false;
             }
-            
-        }
-
-    },
-    created(){
-        const date = new Date();
-        const day = date.getDate();
-        const month = date.getMonth();
-        const year = date.getFullYear();
-        this.dateNow =  day + "/" + month + "/" + year;
+        },
     },
     methods:{
         checkForm:function(e){
```

Comment:
- i think you can write directly checkForm(e){
- i think is more understandable use "event" instead "e"

Reply:
Ok, change resolved